recipes/scippnexus/recipe.yaml

```diff
@@ -0,0 +1,47 @@
+context:
+  name: scippnexus
+  version: 24.11.1
+  python_min: "3.10"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/scippnexus-${{ version }}.tar.gz
+  sha256: bb89a21e843b8faaac0221b6bf15978597737d3040043161e7e396057573722c
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools >=68
+    - setuptools-scm >=8.0
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - python-dateutil
+    - scipp >=24.02.0
+    - scipy
+    - h5py
+
+tests:
+  - python:
+      imports:
+        - scippnexus
+      pip_check: false
```

Comment:
Why?

Reply:
pip check can not detect scipp, our hunch is because [scipp](https://github.com/conda-forge/scipp-feedstock) doesn't add python package metadata https://github.com/scipp/scipp/issues/3649 (hopefully should be resolved soon and we can run pip check for packages that depend on scipp on conda-forge).

Reply:
@chrisburr let me know if it's absolutely required to run pip check here :)